### ISSUE TITLE
feature=user_favorites is not working

### ISSUE THREAD

**Issue by rmandvikar** [500px/api-documentation#172]
This used to work. But now I see that new favorites are not in the json. Is it deprecated?

https://api.500px.com/v1/photos?feature=user_favorites&username=-{username}&consumer_key={consumer-key}

If deprecated, how do I get a user's favorites now?

**Comment by HayoBaan** [comment]
I think you should now go via the user's Galleries. There will be a gallery called Favorites which you should be able to use.

**Comment by rmandvikar** [comment]
If someone favorites a photo, it does not get added to his/her favorites. The api link above does not show the newly faved photo in json but the existing favorites are present. There is a separate process to add a photo to your own gallery. So how do I get user's favorites (existing ones and newly faved photos) using api?

**Comment by moc426** [comment]
You can't add photos as favorites anymore, you can only like/unlike a photo. You should be able to add to the favorites gallery using galleries endpoints.

**Comment by rmandvikar** [comment]
What's the url for that? This is not giving me correct results:
https://api.500px.com/v1/photos?feature=galleries&username={username}&consumer_key={consumer-key}

**Comment by moc426** [comment]
To get images from gallery -> https://github.com/500px/api-documentation/blob/master/endpoints/galleries/GET_galleries_id_items.md

To add images to gallery -> https://github.com/500px/api-documentation/blob/master/endpoints/galleries/PUT_galleries_id_items.md

**Comment by rmandvikar** [comment]
I saw those links. Please give me an url for example request.

**Comment by moc426** [comment]
https://api.500px.com/v1/users/{user_id}/galleries/{gallery_id}/items?consumer_key={consumer-key}

So replace user_id, gallery_id is favorites in this case and your consumer key. Should work.

**Comment by rmandvikar** [comment]
That does not work. I tried below and I get this error for all. 

error: "You have not provided a valid user ID.",
status: 400

https://api.500px.com/v1/users/tchebotarev/galleries?consumer_key={key}

https://api.500px.com/v1/users/tchebotarev/galleries/favorites?consumer_key={key}
https://api.500px.com/v1/users/tchebotarev/galleries/favorites/items?consumer_key={key}

**Comment by moc426** [comment]
User id is a number, not the username.

**Comment by rmandvikar** [comment]
I see. But that still does not solve my issue. Is there an api endpoint to get a user's liked photos now?

**Comment by cdmicacc** [comment]
No, there is no API to get a list of liked photos.  @moc426 is correct otherwise in that the Favorites API is deprecated and you should use the Galleries API going forward.
